tpls/.clang-format

```diff
@@ -0,0 +1,3 @@
+#Official Tool: clang-format version 8.0.0
+DisableFormat: true
```

Comment:
Actually wait.  What does that option make.  Disable in the tpls/ dir?
Why would you bother saying we don't want to sort includes then?

Reply:
Otherwise, it still tries to sort them. I tried without.

Reply:
OK.  Thanks for clarifying.